### ISSUE TITLE
Sync read status between clients

### ISSUE THREAD

**Issue by Spenhouet** [vitordelucca/Reddunt#25]
I did see that some things get synced between my devices. But I'm missing a sync of the read status of posts.
Is that possible?

**Comment by mediaexplorer74** [comment]
> I did see that some things get synced between my devices. But I'm missing a sync of the read status of posts. Is that possible?

Good idea... but... a few years ago... I noticed that Jul, 1 2023 Reddit corporation will start to "brick" (block) Reddit api free using... So... no way ;(
